layouts/partials/banner.html

```diff
@@ -0,0 +1,11 @@
+{{/* When a banner is dismissed, its `name` property is recorded in localStorage to
+prevent it from being shown again, so be sure to name new banners uniquely to prevent them
+from being suppressed. */}}
+<pulumi-banner name="cloud-engineering-summit" dismissible>
+    <div class="content">
+        <span class="summary">Learn from the experts at the Cloud Engineering Summit, October 7&ndash;8.</span>
+        <span class="cta">
+            <a href="https://cloudengineeringsummit.com/" target="_blank" rel="noopener noreferrer">Register for free</a>
```

Comment:
We are going to want to add UTM parameters here since we `cloudengineeringsummit.com` is an external domain of ours. Something like `https://cloudengineeringsummit.com/?utm_campaign=summit&utm_source=pulumi.com&utm_medium=top-banner`

It is important to note that we will only do this for external links if we are linking internally (pulumi.com/) then we will not use UTM parameters to avoid overwriting attribution data.

Reply:
Cool. Do you want to add them here, then, so they'll be correct when we merge?

Reply:
cc @isaac-pulumi